components/cam/bld/namelist_files/use_cases/20TR_E3SMv1_superfast_ar5-emis.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0"?>
+<namelist_defaults>
+
+<!-- Set default output options for CMIP6 simulations -->
+<cosp_lite>.true.</cosp_lite>
+
+<!-- Solar constant from CMIP6 input4MIPS -->
+<solar_data_file>atm/cam/solar/Solar_1850-2299_input4MIPS_c20181106.nc</solar_data_file>
+<solar_data_type>SERIAL</solar_data_type>
+
+<!-- GHG values from CMIP6 input4MIPS -->
+<bndtvghg>atm/cam/ggas/GHG_CMIP-1-2-0_Annual_Global_0000-2014_c20180105.nc</bndtvghg>
+<scenario_ghg>RAMPED</scenario_ghg>
```

Comment:
In the test of this 20TR compset, the atm_in file has:
```
bndtvghg		= '/compyfs/inputdata/atm/cam/ggas/GHG_CMIP-1-2-0_Annual_Global_0000-2014_c20180105.nc'
 co2vmr		= 0.000001e-6
 scenario_ghg		= 'RAMPED'
```
Do we need to add other GHGs, such as CFCs, N2O, etc? If not, how the other GHGs are specified?

Reply:
The concentrations for the GHGs are time varying and will be read out of the data file.

I am not sure what the co2vmr is doing there. It is probably because of the way that the build system handles CO2.  My guess is that it is left in because CO2 is handled separately to ensure consistent concentrations between atm, land, and the other components.   I think it will be ignored by the code which will use the file values instead.  This can be checked by looking at the co2vmr in the output files.